docker/script/bootstrap.sh

```diff
@@ -1,51 +1,65 @@
 #!/bin/sh
 
 set -e
+set -x
```

Comment:
Did you intend to leave this in? The logging that comes from this is great for debug but can be overwhelming for prod applications.

Reply:
Ideally they're using local-runner for debugging and not prod, and this reflects what we have in MWAA